src/js/components/molecules/tooltip-toggle/tooltip-toggle.ts

```diff
@@ -37,45 +40,44 @@ export class TooltipToggle extends Vue implements TooltipToggleProps {
         this.visible = true;
     }
 
-    toggle(): void {
+    showTooltip():void {
```

Comment:
If it is a showTooltip function it shoudn't do a hide, now it is a toggleTooltip function.